src/client/components/landing/_base.js

```diff
@@ -0,0 +1,325 @@
+import {extend, mobileCheck, q, color2Hex} from './helpers.js'
+// const DEBUGMODE = window.location.toString().indexOf('VANTADEBUG') !== -1
+
+const win = typeof window == 'object'
+let THREE = (win && window.THREE) || {}
+if (win && !window.VANTA) window.VANTA = {}
+const VANTA = (win && window.VANTA) || {}
+VANTA.register = (name, Effect) => {
+  return VANTA[name] = (opts) => new Effect(opts)
+}
+VANTA.version = '0.5.4'
+
+export {VANTA}
+
+// const ORBITCONTROLS = {
+//   enableZoom: false,
+//   userPanSpeed: 3,
+//   userRotateSpeed: 2.0,
+//   maxPolarAngle: Math.PI * 0.8, // (pi/2 is pure horizontal)
+//   mouseButtons: {
+//     ORBIT: THREE.MOUSE.LEFT,
+//     ZOOM: null,
+//     PAN: null
+//   }
+// }
+// if (DEBUGMODE) {
+//   extend(ORBITCONTROLS, {
+//     enableZoom: true,
+//     zoomSpeed: 4,
+//     minDistance: 100,
+//     maxDistance: 4500
+//   })
+// }
```

Comment:
What do these do? can they be deleted?

Reply:
I think that's for the other animations offered by the library? _base.js was a dependancy that I needed to download from the repo to get the animation working.

Reply:
> There's only one issue really, Im not sure why we are including the entire source code for Vanta (_base.js and vanta.globe.js) is there a particular reason why we arent using the `vanta.globe.min.js` instead? thinking 

My bad, I didn't realize they had a min file for vanta globe! I'll commit something that uses that instead.